src/App.js

```diff
@@ -1,11 +1,109 @@
-import './App.css';
+import { Routes, Route } from "react-router-dom";
+import CategoryScroller from "./components/CategoryScroller/CategoryScroller";
+import LandingPage from "./components/LandingPage/LandingPage";
+import NavBar from "./components/NavBar/NavBar";
+import WatchPage from "./components/WatchPage/WatchPage";
+import { useLikedHistoryWatchLater } from "./Context/liked-history-watchLater-context";
+import LikedHistoryWatchLaterPage from "./components/LikedHistoryWatchLaterVideosPage/LikedHistoryWatchLaterVideosPage";
+import ClearHistoryBtn from "./components/ClearHistoryBtn/ClearHistoryBtn";
+import NoVideosFoundPage from "./components/NoVideosFoundPage/NoVideosFoundPage";
+import SearchGrid from "./components/SearchGrid/SearchGrid";
 
 function App() {
+  const { state } = useLikedHistoryWatchLater();
+
   return (
-    <div className="App">
-      
-    </div>
+    <>
+      <NavBar />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <>
+              <CategoryScroller />
+              <LandingPage />
+            </>
+          }
+        />
+        <Route path="/:searchQuery" element={
+          <>
+            <SearchGrid />
+          </>
+        } />
+        <Route
+          path="/liked"
+          element={
+            <>
+              {state.likedArr.length === 0 ? (
+                <NoVideosFoundPage />
+              ) : (
+                <LikedHistoryWatchLaterPage
+                  actionTypeOfCross="REMOVE_FROM_LIKED"
+                  arrayToBeMapped={state.likedArr}
+                  cross
+                />
+              )}
+            </>
+          }
+        />
+        <Route
+          path="/watch-later"
+          element={
+            <>
+              {state.watchLaterArr.length === 0 ? (
+                <NoVideosFoundPage />
+              ) : (
```

Comment:
Can use switch case but it's ok as logic is small

Reply:
As there are only two things, i think ternary operator is fine